Added HTTPS to some included scripts

BenoitGour opened the pull request.

BenoitGour committed:
added https to included scripts

BenoitGour committed:
href to src

BenoitGour committed:
update

BenoitGour committed:
padding

BenoitGour committed:
src to href

BenoitGour committed:
src to href

BenoitGour committed:
Merge remote-tracking branch 'origin/master'

BenoitGour committed:
href to src

BenoitGour committed:
styles.css

BenoitGour committed:
fix1

BenoitGour committed:
removed refs to package.json

BenoitGour committed:
skip package route

BenoitGour committed:
skip package route

BenoitGour committed:
Merge remote-tracking branch 'origin/master'

BenoitGour committed:
Merge remote-tracking branch 'origin/master'

BenoitGour committed:
Merge remote-tracking branch 'origin/master'

BenoitGour committed:
dont skip first node

BenoitGour committed:
info tab

BenoitGour committed:
info init

BenoitGour committed:
info fix

BenoitGour committed:
fix

BenoitGour committed:
debug

BenoitGour committed:
fix

BenoitGour committed:
typo fix

BenoitGour committed:
typo fix

BenoitGour committed:
Merge remote-tracking branch 'origin/master'

BenoitGour committed:
fix

BenoitGour committed:
fix

BenoitGour committed:
fix

BenoitGour committed:
Merge remote-tracking branch 'origin/master'

BenoitGour committed:
test

BenoitGour committed:
test

BenoitGour committed:
info

BenoitGour committed:
test

BenoitGour committed:
test

BenoitGour committed:
test

BenoitGour committed:
card format

BenoitGour committed:
typo

BenoitGour committed:
details

BenoitGour committed:
card

BenoitGour committed:
card

BenoitGour committed:
card

BenoitGour committed:
card format

BenoitGour committed:
card format

BenoitGour committed:
Merge remote-tracking branch 'origin/master'

BenoitGour committed:
fix

BenoitGour committed:
style

BenoitGour committed:
pre

BenoitGour committed:
page title

BenoitGour committed:
page title

BenoitGour committed:
getPageTitle

BenoitGour committed:
debug

BenoitGour committed:
debug

BenoitGour committed:
title

BenoitGour committed:
title

BenoitGour committed:
title

BenoitGour committed:
title

BenoitGour committed:
title

BenoitGour committed:
title

BenoitGour committed:
title

BenoitGour committed:
title

BenoitGour committed:
title

BenoitGour committed:
title

BenoitGour committed:
title

BenoitGour committed:
version lable

BenoitGour committed:
version label position

BenoitGour committed:
fix

BenoitGour committed:
fix

BenoitGour committed:
fix

smlee commented:
@BenoitGour I will take a look and get the merge conflict fixed later tonight. Sorry for the delay and thank you for your contribution

BenoitGour committed:
label fix

BenoitGour committed:
label position

BenoitGour committed:
label style

BenoitGour committed:
font size

BenoitGour committed:
page title


page title


getPageTitle


debug


debug


title


title


title


title


title


title


title


title


title


title


title


version lable


version label position


fix


fix


fix


label fix


label position


label style


font size

BenoitGour committed:
Merge remote-tracking branch 'origin/master'

BenoitGour committed:
endpoint font size

BenoitGour committed:
info tab location

BenoitGour commented:
@smlee oh sorry, please close ignore this pull request,  I did not realise my other modifications/experimentation were automaticaly added to this pull request...
If you are however interested  here is what i wanted to modify:  https://github.com/smlee/pathfinder-ui/pull/81/commits/9e0dc7b590df848f08f295869c2695eb12422184
